### ISSUE TITLE
Abnormal circulating Serum Amyloid P Component concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6006]
**Preferred term label:** Abnormal circulating Serum Amyloid P Component concentration

**Synonyms** Abnormal circulating SAP

**Definition (free text, please give PubMed ID)** Concentration of serum amyloid P component in the blood is outside the normal range., PMID: 28098450

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Serum Amyloid P Component concentration

**Synonyms** Decreased circulating SAP

**Definition (free text, please give PubMed ID)** Concentration of serum amyloid P component in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Serum Amyloid P Component concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Serum Amyloid P Component concentration

**Synonyms** Increased circulating SAP

**Definition (free text, please give PubMed ID)** Concentration of serum amyloid P component in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Serum Amyloid P Component concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
SAP (PTX2) is a member of the pentraxin family of proteins that includes C-reactive protein (CRP; PTX1) and pentraxin-3 (PTX3). SAP is made by hepatocytes and secreted into the blood [PMID:30459752]

**Comment by pnrobinson** [comment]
I am not sure whether blood testing to SAP is commonly done, https://www.mayocliniclabs.com/test-catalog/Clinical+and+Interpretive/70549

**Comment by pnrobinson** [comment]
To the best of my knowledge, there is no clinical test for circulating amyloid P, with a normal range. There is no LOINC term for this that I can find.
